tests/bootstrap.php

```diff
@@ -96,6 +96,11 @@
     Artisan::call('migrate:fresh', []);
     Artisan::call('db:seed', ['--class' => 'AnonymousUserSeeder']);
 
+    // In \ProcessMaker\Console\Commands\MigrateFresh:35
+    // The migrations table is being manually created to resolve the bug described below.
+    // https://github.com/ProcessMaker/processmaker/pull/4537
```

Comment:
@paulosakamoto I appreciate the comments but I think we should remove them so that we don't overwhelm our code with references to tickets. If we ever need to trace back this addition to the codebase we can use `git blame` to do so. Thank you!

Reply:
Done.